### ISSUE TITLE
"unable to find a password field"

### ISSUE THREAD

**Issue by Neophema** [pfn/passifox#500]
ChromeIpass seems to have an issue with filling in a Webex access window. Something I do 7 ot 8 times a day. 

Pressing Ctrl-Shift-P gets me this
![image](https://cloud.githubusercontent.com/assets/15657720/15029744/9b914d36-1214-11e6-9461-d69b7e2cbb3c.png)

If I use "choose own credential fields for this page" I can choose the uername but no password field is available: 
![image](https://cloud.githubusercontent.com/assets/15657720/15029882/360dbd90-1215-11e6-81c0-b6e2a3d17192.png)

**Comment by pfn** [comment]
Yes, I don't think you can force a plain text field to be a password field
